### PR TITLE
crystal: add page

### PR DESCRIPTION
**Pull request by aa-tan**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

**Comment by aa-tan** [comment]
Any idea why commit 8a49735 still did not pass the style guidelines? I ended up recommitting after deleting and rewriting line 17. Possibly some invisible character?

**Comment by aa-tan** [comment]
Disregard that, there was a space at the end of the line. Also, apologies that the two commits that followed are not in line with the contribution guidelines, let me know if this PR should be deleted, I can submit one again after making the proper changes if necessary.

**Comment by mfrw** [comment]
@aa-tan Thank you for your contribution. No, you don't have to send in a new PR. We will try to work with this :)
Don't worry about the mistakes, it happens to all of us :)
Welcome aboard!

File: pages/common/crystal.md [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 `crystal play`
 
-- Create a project directory for a crystal application: 
+- Create a project directory for a crystal application:
 
 `crystal init app {{name of application}}`
```
Comment (mfrw) [inline pages/common/crystal.md]: Can we have a concrete example here instead of a generic one?

File: pages/common/crystal.md [hunk 2]
```diff
@@ -14,7 +14,7 @@
 
 `crystal play`
```
Comment (mfrw) [inline pages/common/crystal.md]: Are there any other common usages that we could add to this page?

File: pages/common/crystal.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+`crystal build {{path/to/file.cr}}`
+
+- Start a local interactive server for testing the language:
+
+`crystal play`
+
+- Create a project directory for a crystal application:
+
+`crystal init app {{name of application}}`
```
Comment (mfrw) [inline pages/common/crystal.md]: Can we have a concrete example here instead of a generic one?

Comment (pxgamer) [reply pages/common/crystal.md]: Just to note, this should be changed to token syntax like `name_of_application` (or `application_name`). Unless it gets changed to a concrete example. :thinking:

File: pages/common/crystal.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
+# crystal
+
+> Crystal language tool for compiling.
```
Comment (agnivade) [inline pages/common/crystal.md]: We should make this consistent with other pages like `go`. Like - `Tool for managing crystal source code`

File: pages/common/crystal.md (hunk 3 above)
Comment (aa-tan) [reply pages/common/crystal.md]: Would `crystal init app my_blog` serve as a more concrete example?

File: pages/common/crystal.md (hunk 4 above)
Comment (aa-tan) [reply pages/common/crystal.md]: Fixed with b5de468

**Comment by agnivade** [comment]
I updated application name myself. Made it consistent with other pages like `npm`, `dpkg`, `pacman` where we have package_name, module_name etc.

**Pull request merged**